platform/wm/rhodes/MainWindow.cpp

```diff
@@ -1241,8 +1241,11 @@ LRESULT CMainWindow::OnExecuteJSCommand(WORD /*wNotifyCode*/, WORD /*wID*/, HWND
 {
     TNavigateData* nd = (TNavigateData*)hWndCtl;
     if (nd) {
-        LPTSTR wcurl = (LPTSTR)(nd->url);
-        if (wcurl) 
+        wchar_t wcurl[2046];
+	memset(wcurl,'0',2046);
```

Comment:
Well in that case... I will be creating the size dynamically.. I will modify the same.